app/models/experimentation/notifications_experiment.rb

```diff
@@ -259,14 +260,16 @@ def reporting_data(patient, date)
       latest_scheduled_appointment = patient.latest_scheduled_appointment
       assigned_facility = patient.assigned_facility
       registration_facility = patient.registration_facility
+      expected_return_date = latest_scheduled_appointment&.remind_on || latest_scheduled_appointment&.scheduled_date
+      expected_return_date_utc_timestamp = expected_return_date&.to_time(:utc)
```

Comment:
`to_time` is for converting string instances to date. If the intention is to convert to utc timezone, it might be better to use `.utc` instead?
```suggestion
      expected_return_date_utc_timestamp = expected_return_date&.utc
```

Reply:
Do we want to convert a `Date` object into a `DateTime` object, and then persist to DB?
If yes, then this implementation is correct.